packages/sanity/src/core/changeIndicators/ElementWithChangeBar.styled.tsx

```diff
@@ -6,54 +6,54 @@ interface ThemeContext {
 }
 
 interface RootProps {
-  focus: boolean
-  hover: boolean
```

Comment:
These were removed as the styled component wasn't doing anything with these values